mirror_maker.go

```diff
@@ -17,9 +17,25 @@ package go_kafka_client
 
 import (
 	"fmt"
+	avro "github.com/stealthly/go-avro"
 	"hash/fnv"
+	"time"
 )
 
+var TimingField = &avro.SchemaField{
```

Comment:
This should evolve to be the same as LogLine proto (but in Avro) with a new timing structure that is a record key/value pair. You should capture for each atomic time for the event also what the serde time was too